recipes/newrelic/infrastructure/alerts/golden.yml

```diff
@@ -0,0 +1,348 @@
+name: alerts-golden-signal
+displayName: Golden Signal Alerts
+description: New Relic install recipe for golden signal alerts
+repository: https://github.com/newrelic/newrelic-cli
+
+installTargets:
+  - type: host
+    os: linux
+
+keywords:
+  - Alerts
+  - Golden
+
+processMatch: []
```

Comment:
Correct, it will run for any guided install, there is no downside to have it run, it creates up to 1 policy which covers all host and applications reporting to newrelic.